src/pages/AllActivitiesPage.js

```diff
@@ -0,0 +1,130 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import Header from '../components/Header';
+import Main from '../components/Main';
+import MappedPlaceholderPictures from '../components/MappedPlaceholderPictures.js';
+import Navigation from '../components/Navigation';
+import Picture from '../components/Picture';
+import Searchbar from '../components/Searchbar';
+
+import newIcon from '../images/newIcon.svg';
+import nextIcon from '../images/nextIcon.svg';
+
+export default function AllActivitiesPage({ activities }) {
+  const [searchInput, setSearchInput] = useState('');
+
+  const searchInputHandler = dataInput => {
+    const lowerCase = dataInput.target.value.toLowerCase();
+    setSearchInput(lowerCase);
+  };
+
```

Comment:
hmmh not sure but why did you call it dataInput and not event?

Reply:
its for me to make it more clear, however, normally it should be event - you are right :)